src/woo_publications/utils/query.py

```diff
@@ -0,0 +1,20 @@
+from drf_spectacular.contrib.django_filters import DjangoFilterExtension
+
+from .utils import underscore_to_camel
+
+
+class CamelizeFilterExtension(DjangoFilterExtension):
+    priority = 1
+
+    def get_schema_operation_parameters(self, auto_schema, *args, **kwargs):
+        """
+        camelize query parameters
+        """
+        parameters = super().get_schema_operation_parameters(
```

Comment:
to address the coverage remarks, I'm okay with putting a `# pragma: no cover` on the class definition. We have CI checks that generate the API schema that ensure this behaves as expected, so we don't need unit tests for this.